### PR TITLE
SUBMARINE-385. [k8s-e2e] Create k8s e2e test module

### PR DESCRIPTION
**Pull request by liuxunorg**
### What is this PR for?
1. Add new travis job, create k8s by Kind & deploy submairne in k8s
1. Support developers to use kind to create k8s cluster in local / travis
2. depoly submarine in kind cluster
3. By calling the submarine's rest interface, determine whether the submarine is deployed correctly in k8s.  like this : curl `http://127.0.0.1/api/v1/cluster/nodes`

### What type of PR is it?
Feature

### Todos
* [ ] - Support tensorflow in k8s e2e test
* [ ] - Support pytorch in k8s e2e test

### What is the Jira issue?
* https://issues.apache.org/jira/browse/SUBMARINE-385

### How should this be tested?
* https://travis-ci.org/liuxunorg/submarine/builds/650766597

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-test/test-k8s/pom.xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <version>${slf4j.version}</version>
```
Comment (yuanzac) [inline submarine-test/test-k8s/pom.xml]: slf4j.version can be deleted and use the default version

File: submarine-test/test-k8s/pom.xml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <version>${testng.version}</version>
```
Comment (yuanzac) [inline submarine-test/test-k8s/pom.xml]: testing.version can be deleted and use the default version

File: submarine-test/test-k8s/pom.xml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        <exclusion>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>commons-httpclient</groupId>
+      <artifactId>commons-httpclient</artifactId>
+      <version>${commons-httpclient.version}</version>
```
Comment (yuanzac) [inline submarine-test/test-k8s/pom.xml]: version tag can be deleted

File: submarine-test/test-k8s/pom.xml [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>${commons-lang.version}</version>
```
Comment (yuanzac) [inline submarine-test/test-k8s/pom.xml]: version can be deleted.

File: submarine-test/test-k8s/pom.xml [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>javax.ws.rs-api</artifactId>
+      <version>${javax.ws.rsapi.version}</version>
```
Comment (yuanzac) [inline submarine-test/test-k8s/pom.xml]: version can be deleted

**Comment by liuxunorg** [comment]
Will merge if no more comments.

**Pull request closed without merging**